lumigator/frontend/src/components/experiments/LExperimentForm.vue

```diff
@@ -231,30 +231,50 @@ async function handleRunExperimentClicked() {
     system_prompt: experimentPrompt.value || defaultPrompt.value,
   }
 
-  const workflows = await experimentsService.createExperimentWithWorkflows(
+  const [experimentId, workflowResults] = await experimentsService.createExperimentWithWorkflows(
```

Comment:
nit: `workflowResults` can be misinterpreted as the real workflow results

```suggestion
  const [experimentId, createWorkflowResponses] = await experimentsService.createExperimentWithWorkflows(
```